### PR TITLE
Feature/bc 999 update status pages

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description

As a user, I want to be directed to the new status page when I click the link "Status" in the footer.

## Links to Tickets or other pull requests

Ticket: https://ticketsystem.hpi-schul-cloud.org/browse/BC-999
Client-PR: https://github.com/hpi-schul-cloud/schulcloud-client/pull/2739 
Server-PR: https://github.com/hpi-schul-cloud/schulcloud-server/pull/3259
Dev-PR: https://github.com/hpi-schul-cloud/dof_app_deploy/pull/82

## Changes

- environment variable set to default (dBildungscloud) and individually set for each instance

## Deployment

ALERT_STATUS_URL: https://status.INSTANCE.de

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by tofixx — COMMENTED** [review]
i don't see the vars correctly set in lens for nuxt

**Comment by virgilchiriac** [comment]
missing ansible config for nuxt

**Comment by MartinSchuhmacher** [comment]
> missing ansible config for nuxt

thx for the hint! was added now

**Review by Metauriel — DISMISSED** [review]

File: src/themes/default/components/legacy/TheFooter.vue [hunk 1]
```diff
@@ -43,6 +43,7 @@
 <script>
 import AuthModule from "@/store/auth";
 import FilePathsModule from "@/store/filePaths";
+import EnvConfigModule from "@/store/env-config";
```
Comment (murat-merdoglu-hpi) [inline src/themes/default/components/legacy/TheFooter.vue:47]: I think that we need to add a test for this file.

File: src/components/legacy/TheFooter.vue [hunk 2]
```diff
@@ -82,8 +83,10 @@ export default {
 					text: this.$t("components.legacy.footer.github"),
 				},
 				{
-					href: "https://status.hpi-schul-cloud.de",
+					href: EnvConfigModule.getStatusLink,
```
Comment (murat-merdoglu-hpi) [inline src/components/legacy/TheFooter.vue:86]: I think we need to add a test for this case.

**Review by Metauriel — APPROVED** [review]

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Pull request merged**